cmd/admin-config-get.go

```diff
@@ -59,10 +60,12 @@ func (u configGetMessage) String() string {
 // JSON jsonified service status Message message.
 func (u configGetMessage) JSON() string {
 	u.Status = "success"
-	statusJSONBytes, e := json.Marshal(u)
+	statusJSONBytes, e := json.MarshalIndent(u, "", "    ")
```

Comment:
`"\t"` can be used instead of bunch of space characters, like
`json.MarshalIndent(u, "", "\t")`

Reply:
done